chart/templates/deployment.yaml

```
@@ -78,6 +78,10 @@ spec:
                   fieldPath: metadata.namespace
             - name: SCRAPE_INTERVAL
               value: '{{ ((.Values.prometheus).scrapeInterval) | default "5s" }}'
+            - name: CREATE_TARGETS
+              value: '{{ .Values.prometheus.createTargets }}'
```

can we change this to
```suggestion
              value: '{{ ((.Values.prometheus).createTargets) | default "true" }}'
```
in order to avoid problem with `helm upgrade ... --reuse-values`?